syntax = "proto3";


package scrcpyx.mgr.v1;

message StartScrcpyServerRequest {
  string did = 1;
  //  args to start the scrcpy server
  repeated string args = 2;
}

message StartScrcpyServerResponse {
  string session_id = 1;
}

service ScrcpyxMgrService {
  rpc StartScrcpyServer(StartScrcpyServerRequest) returns (StartScrcpyServerResponse);
}
